### DISCUSSION TITLE
Possible to change the background on only the main meeting list page

### DISCUSSION THREAD

**Discussion by toddjsaunders** [code4recovery/12-step-meeting-list#653] [Q&A] [answered]
Hi, 

My usual method of changing the background for a single page is via CSS and the page id, eg:

body.page.page-id-XX {
background-color: YYY;
}

But I can't see a page id for the main listing page, only the individual meeting detail pages.

Any help would be greatly appreciated -- thanks!

**Comment by joshreisner** [comment, marked as answer]
hi @toddjsaunders try 
```
body.tsml-meetings {
  background-color: tomato;
}
```

**Reply by toddjsaunders** [reply]
Thanks so much Josh that did the trick!
